src/Halogen/HTML/Properties.purs

```diff
@@ -80,6 +81,7 @@ import DOM.HTML.Indexed.MenuitemType (MenuitemType(..)) as I
 import DOM.HTML.Indexed.OnOff (OnOff(..)) as I
 import DOM.HTML.Indexed.OrderedListType (OrderedListType(..)) as I
 import DOM.HTML.Indexed.PreloadValue (PreloadValue(..)) as I
+import DOM.HTML.Indexed.ScopeValue(ScopeValue)
```

Comment:
```suggestion
import DOM.HTML.Indexed.ScopeValue (ScopeValue(..)) as I
```

I'd suggest echoing the other property-value-imports (like `PreloadValue`, `OrderedListType`, etc. above it). 

By doing this it re-exports the `ScopeValue` constructors (since `module I` is exported), which makes things a bit easier when using the HTML DSL, since you don't have to go hunting for the module to import to provide the value to `HP.scope` then - it'll be available as `HP.ScopeRow` (or whichever) too.